spec/index.spec.ts

```diff
@@ -28,11 +28,11 @@ import * as nock from 'nock';
 nock.disableNetConnect();
 
 import 'mocha';
-import { setup } from '../src/index';
 import './utils.spec';
 import './apps.spec';
 import './cloud-functions.spec';
 import './config.spec';
+import './setup.spec';
```

Comment:
I think since the test is in `setup.spec` it would be good to move the code to be there too instead of being a function in`index.ts` so that everything is consistent and the reader knows where to find the test code for the source file. What do you think?

Reply:
ok made a `setup.ts`